src/test/java/net/sf/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -112,6 +112,12 @@ public void testMonthChecks() {
     }
 
     @Test
+    public void testBibtexkeyChecks() {
+        assertCorrect(createContext("bibtexkey", "Knuth2014"));
```

Comment:
Removed my thumbs up. A separate class for every checker is too much (for evt. one or two entries).  I think, it is better to organize the checker tests by fields (in methods) and not by checker classes.